contributing/contribution_guide.rst

```diff
@@ -0,0 +1,128 @@
+How to contribute ?
+===================
+
+As Akeneo PIM is an open-source project all contribution is very welcome.
+
+Install and configure Git
+-------------------------
+
+Follow this link from git to install https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+
+Configure
+
+.. code-block:: php
+
+    git config --global user.name "Your Name"
+    git config --global user.email you@example.com
+
+Create an account on github https://github.com/
+
+Create a branch
+---------------
+
+.. note::
+
+    New to git, learn with : https://git-scm.com/book/en/v2
+
+First of all, fork Pim Community development version on https://github.com/akeneo/pim-community-dev (click on the Fork button).
+
+Then clone your fork locally
+
+.. code-block:: php
+
+    $ git clone git@github.com:USERNAME/pim-community-dev.git
+
+Add the upstream repository as a remote
+
+.. code-block:: php
+
+    $ cd pim-community-dev
+    $ git remote add upstream git://github.com/akeneo/pim-community-dev.git
+
+Create a new branch
+
+.. code-block:: php
+
+    $ git checkout -b BRANCH_NAME master
+
+.. note::
+
+    How to name your branch ?
```

Comment:
Core team use internal ticket number (Jira), they looks like PIM-1234.
This convention seems ok for contribution which references github issues.